start_docker.sh

```diff
@@ -44,7 +44,7 @@ fi
 # find lowest open port available
 PORT=8888
 
-until [[ $(docker container ls | grep 0.0.0.0:$PORT | wc -l) -eq 0 ]]
+until [[ $(netstat -an | grep LISTEN | grep $PORT | wc -l) -eq 0 ]]
```

Comment:
What does this do?

Reply:
If people are running local Jupyter servers using `docker container ls` doesn't always find them. `netstat -an | grep LISTEN` does the trick more thoroughly.